lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -114,6 +114,8 @@ def init_from_model(self, student, model, course_structure, submissions_scores,
         Load the subsection grade from the persisted model.
         """
         for block in model.visible_blocks.blocks:
+            if block.locator not in set(course_structure.post_order_traversal(start_node=self.location)):
```

Comment:
Also, it's only traversing the subtree whose root is the current subsection, right? Or does `start_node` not do what I think it does?